indexing-service/src/test/java/org/apache/druid/indexing/common/task/batch/parallel/AbstractParallelIndexSupervisorTaskTest.java

```diff
@@ -546,12 +548,17 @@ public ListenableFuture<Void> runTask(String taskId, Object taskObject)
 
     @Override
     public ListenableFuture<Map<String, Object>> taskReportAsMap(String taskId)
+    {
+      return Futures.immediateFuture(null);
```

Comment:
Should this call `getLiveReportsForTask(taskId)`?

Reply:
It was doing that originally but not needed right now as I have added the other method `getLiveReportsForTask()` just below this one.

This is anyway used only in the tests and I plan to fix it back up once I replace the `Map<String, Object>` in the `OverlordClient` with `TaskReport.ReportMap`.

Reply:
Ok, makes sense